cache-config/t3c-check-refs/t3c-check-refs.go

```diff
@@ -115,7 +115,7 @@ func checkConfigLine(line string, lineNumber int, filesAdding map[string]struct{
 				// are assumed to be configuration files and are checked that they
 				// exist in the filesystem at the absolute location in the name
 				// or relative to the ATS configuration files directory.
-				m := regexp.MustCompile(`^*(\.config|\.cfg|\.txt|\.yml|\.yaml|\.lua)+`)
+				m := regexp.MustCompile(`^*(\.config|\.cfg|\.txt|\.yml|\.yaml|\.lua)$`)
```

Comment:
this looks we introduced a typo. (probably me) while we're changing it we should change the beginning to `^.+` regex101 tool doesn't like the `*`

Reply:
Indeed, the beginning of the regex is not valid too and it seems the Go regexp lib is not complaining
I have fixed that too with your suggestion, thanks